config.go

```diff
@@ -418,7 +418,7 @@ func (settings *Settings) SessionFactory() *legacyDB.SessionFactory {
 
 func (s *Settings) GetGithubOauthToken() (string, error) {
 	token, ok := s.Credentials["github"]
-	if ok {
+	if ok || token == "" {
```

Comment:
We will never hit the error case.

Reply:
If the token in the config is set to empty string, we will.